lms/djangoapps/course_api/blocks/tests/test_views.py

```diff
@@ -103,6 +103,7 @@ def verify_response_with_requested_fields(self, response):
             self.assert_in_iff('children', block_data, xblock.has_children)
             self.assert_in_iff('graded', block_data, xblock.graded is not None)
             self.assert_in_iff('format', block_data, xblock.format is not None)
+            self.assert_in_iff('due', block_data, xblock.due is not None)
```

Comment:
It seems the `ToyCourseFactory` doesn't set the `due` field for any of its blocks.  Can you verify?  If not, you should set it on one or more of its blocks to a future date.